plugin/move-less.vim

```diff
@@ -39,22 +56,22 @@ function! MoveLessMode()
     endif
 
     let l:mode = 'initial'
-    while l:result ==? 'j' || l:result ==? 'k' || l:result ==? 'l' || l:result ==? 'h'
+    while l:result ==? g:MoveLess#Mappings['FoldBelow'] || l:result ==? g:MoveLess#Mappings['FoldAbove'] || l:result ==? g:MoveLess#Mappings['FoldAboveAndBelowK1'] || l:result ==? g:MoveLess#Mappings['UnfoldAboveAndBelowK1']
```

Comment:
because of the static mapping i have checked the character upper/lower case pairs `l/L` and `h/H` only once but incasesensetive.
you have now substituted it with the custom mapping. but now you have to test for all keys separately because if someone choose instead of `h` and `H` -> `h` and `q` for up and down folding then q would quit the move less mode.

that's true for both if expressions where the key are checked (that the first one don't have the stop key is still correct)

Reply:
do you mean putting checks for overlapping keys? now that you have to pass the whole map explicitly and set each key manually, I think there is less of a chance of this happening. I've found that most plugins don't play around misconfiguration.

Reply:
no i mean you need to check with `==#` which is case insensetive and you need to take all keys into account which the original checking was included.

So the Original line `l:result ==? 'j' || l:result ==? 'k' || l:result ==? 'l' || l:result ==? 'h'`
was true for the keys: `j`,`J`,`k`,`K`,`l`,`L`,`h`,`H`
which now have 8 counterparts at your configuration map.
but now we have to work with case insensitive checks and take every one explicit into account.
Because we don't know if the user will choose lower upper pairs of the same character for an action and the counterpart of the action like it was before.